recipes/highiq/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "highiq" %}
+{% set version = "2025.11.10" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/highiq-{{ version }}.tar.gz
+  sha256: 3607795158a7c43ffa039000833f010c258823982c747c80912e02502353284e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >={{ python_min }}
+    - setuptools >=61.0
+    - wheel
+    - pip
+  run:
+    - python >={{ python_min }}
+    - numpy
+    - scipy
+    - xarray
+    - pint
+    - netcdf4
+
+test:
+  imports:
+    - highiq
+  commands:
+    - pip check
+  requires:
+    - pip
```

```suggestion
    - pip
    - python =={{ python_min }}
```

This would resolve the other warning.